### PR TITLE
fix all TS build errors

### PR DESCRIPTION
**Pull request by Cosmin-Mare**

### CODE REVIEW

**Review by JosiasAurel — APPROVED** [review]
lgtm

File: src/pages/api/games/start-saving.ts [hunk 1]
```diff
@@ -55,15 +57,19 @@ export const post: APIRoute = async ({ request, cookies }) => {
...
+		await updateDocument("games", gameId, { 
 			tutorialName: tutorialName ?? "",
 			modifiedAt: Timestamp.now(),
 		});
+
+		await setDocument('daily-edits', `${trackingId}-${trackingDate}`, {
+			type: trackingType,
+			id: trackingId,
+			date: Timestamp.now()
+		});
```
[the remark this answers is not in the document]
Comment (Cosmin-Mare) [reply src/pages/api/games/start-saving.ts:69]: oh actually nvm, this was intended. I deleted it by mistake

**Comment by Cosmin-Mare** [comment]
@JosiasAurel  answered ur comments

**Review by JosiasAurel — APPROVED** [review]
lgtm

**Pull request merged**